lib/pharos/phases/configure_master.rb

```diff
@@ -310,14 +325,39 @@ def upload_authentication_token_webhook_certs(webhook_config)
         @ssh.file(PHAROS_DIR + '/token_webhook/key.pem').write(File.open(File.expand_path(webhook_config[:user][:client_key]))) if webhook_config[:user][:client_key]
       end
 
+      def secrets_encryption_exist?
+        logger.debug { "Checking if secrets encryption is already configured" }
```

Comment:
Missing ` ...`? And should it say something if already configure?

Reply:
fixed